tox.ini

```diff
@@ -21,10 +20,10 @@ usedevelop = true
 commands = make test
 whitelist_externals = make
 deps =
-    dj111: Django>=1.11a1,<2.0
```

Comment:
There's no need to remove a1. IMO that just makes it less obvious how to test with pre-releases in the future.

Reply:
Done